src/styles/blocks/header.scss

```diff
@@ -0,0 +1,78 @@
+.main {
+  &__catalog {
+    margin: 0 auto;
+    padding: 50px $catalog-side-padding 72px;
+  }
+}
+
+.header,
+.nav__list,
+.nav__link {
+  display: flex;
+}
```

Comment:
it's better to specify display: flex on each item separately, for more clarity

Reply:
agree